models/AmazonS3.cfc

```diff
@@ -325,12 +337,12 @@ component accessors="true" singleton {
 		if ( results.error ) {
 			throw( message = "Error making Amazon REST Call", detail = results.message );
 		}
+		// Should this return whatever comes from AWS? It seems like hardcoding a potentially wrong answer is not a good idea.
+		// if ( len( results.response.LocationConstraint.XMLText ) ) {
```

Comment:
Restore this conditional, I think.  Otherwise it's a breaking change that could effect how consumers are using the SDK.

Reply:
Done.